tests/unit_test/apis/impl/controller_test.py

```diff
@@ -1187,14 +1187,16 @@ def test_task_only_exit_when_wait_time_after_min_received(
             )
 
         wait_time = 0
+        expected_standing_tasks = 1
         while wait_time <= wait_time_after_min_received:
-            assert controller.get_num_standing_tasks() == 1
+            assert controller.get_num_standing_tasks() == expected_standing_tasks
             for client in clients:
                 task_name_out, client_task_id, data = controller.process_task_request(client, fl_ctx)
                 assert task_name_out == ""
                 assert client_task_id == ""
             time.sleep(1)
             wait_time += 1
+            expected_standing_tasks = 0
```

Comment:
instead of changing things here, suggest changing 
line 1155 to `controller, fl_ctx, clients = self.setup_system(num_of_clients=min_responses+1)`

Reply:
It is not the same thing
The changes proposed here is changing the semantic of this specific test.
The changes proposed here will change the test semantic to test "task exit when all the responses received".

I suggest changing line 1155 to `controller, fl_ctx, clients = self.setup_system(num_of_clients=min_responses+1)`
So we can keep the test to test the same thing as it stated `test_task_only_exit_when_wait_time_after_min_received`